### PR TITLE
Changing the export methode for diagrams.

### PR DESCRIPTION
**Pull request by staudtMarius**
Chsnging the export methode for diagrams using "http://www.plantuml.com/plantuml/proxy?cache=no&src="

### CODE REVIEW

File: docs/readthedocs/models/bm_model.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ The BM model is part of the SIMONA Simulation framework and represented by an ag
 
 ### Parameters
 
-![](../_static/figures/models/bm_model/bm_parameters.png)
+![](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/ie3-institute/simona/dev/docs/readthedocs/_static/figures/models/bm_model/bm_parameters.png)
```
Comment (danielfeismann) [inline docs/readthedocs/models/bm_model.md:10]: Please check, that only uml diagrams send through the proxy of plantuml. Here the input of a .png leads to an error.

**Review by danielfeismann — CHANGES_REQUESTED** [review]

File: docs/readthedocs/models/bm_model.md (hunk 1 above)
Comment (staudtMarius) [reply docs/readthedocs/models/bm_model.md]: I checked all the diagrams and fixed the problems.

**Review by danielfeismann — APPROVED** [review]
Looks good, thank you!

**Review by sebastian-peter — APPROVED** [review]
Looks good indead!

**Pull request merged**